projects/admin/src/app/record/detail-view/library-detail-view/location/location.component.html

```diff
@@ -50,3 +50,6 @@
     </div>
   </div>
 </ng-container>
+<ng-template #loading>
+  <i class="fa fa-spinner"></i>
+</ng-template>
```

Comment:
We could use a global spinner like https://www.npmjs.com/package/ngx-spinner, instead of redefining it in each templates. But it's out of the scope of this PR.